lib/babel-preset.js

```diff
@@ -0,0 +1,21 @@
+exports.buildPreset = buildPreset;
```

Comment:
Please at the end of the file and doesn't it work with ES2015 Module Syntax + `.babelrc` in defaults aswell (not 💯 )